build.sbt

```diff
@@ -304,7 +307,18 @@ lazy val frontend: Project = project
     (Compile / run / mainClass) := Some("bloop.Cli"),
     (Compile / run / bloopMainClass) := Some("bloop.Cli"),
     buildInfoPackage := "bloop.internal.build",
-    buildInfoKeys := bloopInfoKeys(nativeBridge04, jsBridge06, jsBridge1),
+    buildInfoKeys := List[BuildInfoKey](
+      Keys.organization,
+      build.BuildKeys.bloopName,
+      Keys.version,
+      Keys.scalaVersion,
+      nailgunClientLocation,
+      "zincVersion" -> Dependencies.zincVersion,
+      "bspVersion" -> Dependencies.bspVersion,
+      "nativeBridge04" -> (nativeBridge04Name + "_" + Keys.scalaBinaryVersion.value),
```

Comment:
Previously,  we would calculate the artifact name using sbt, but this started to cause cycles when trying to make it work under new buildinfo. Instead I decided to remove everything that is not used and just figure out the artifact name using a small heuristic.

Reply:
Huh, I would have thought there was a `Key` that could give this to you, but I can't see one either.

Reply:
yeah, I couldn't find it either, so decided on a heuristic